src/chanjo2/meta/handle_load_intervals.py

```diff
@@ -28,15 +29,11 @@
 LOG = logging.getLogger("uvicorn.access")
 
 
-async def parse_resource_lines(url) -> Tuple[List[List[str]], List[str]]:
-    """Returns header and lines of a downloaded resource."""
-    all_lines: List[str] = "".join(
-        [i.decode("utf-8") async for i in stream_resource(url=url)]
-    ).split("\n")
-    all_lines = [line.rstrip("\n") for line in all_lines]
-    resource_header: str = all_lines[0]
-    resource_lines: List[str] = all_lines[1:-2]  # last 2 lines don't contain data
-    return resource_header.split("\t"), resource_lines
+def resource_lines(url) -> Iterator[str]:
```

Comment:
```suggestion
def read_resource_lines(url) -> Iterator[str]:
```
What is the url type?

Reply:
str, I'll add it!